jerry-core/parser/js/js-parser-expr.c

```diff
@@ -113,6 +113,30 @@ parser_check_invalid_assign (parser_context_t *context_p) /**< context */
   }
 } /* parser_check_invalid_assign */
 
+#if ENABLED (JERRY_ES2015)
+/**
+ * Check and throw an error if the "new.target" is invalid as a left-hand side expression.
+ */
+static void
+parser_check_new_target_lhs (parser_context_t *context_p,
+                             cbc_opcode_t opcode) /**< parser context */
+{
+  JERRY_ASSERT ((opcode == CBC_ASSIGN
+                && (context_p->token.type == LEXER_ASSIGN
+                    || LEXER_IS_BINARY_LVALUE_TOKEN (context_p->token.type)))
+                || opcode == CBC_PRE_INCR
```

Comment:
How about a rage check for the opcode? `opcode >= PRE_INC & opcode <= POST_DECR`?

Reply:
Fixed